cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -103,3 +103,24 @@ def test_projects_dashboard_icons(provider, appliance, soft_assert, container_pr
                 containers_cls.__name__, api_values[containers_cls], statusbox_value
             )
         )
+
+
+def test_project_has_provider(appliance, soft_assert):
```

Comment:
REQUIRED: I think you need to add provider fixture here. If it is absent and you try to run this test only, it is possible that provider won't be added.

Reply:
@izapolsk At the beginning of this module there is a call for ```setup_provider```, I am not sure but I think that provider is added when any test is triggered on this suite, right?

https://github.com/shalomnaim1/integration_tests/blob/1871af20e6849deb2edc5491795ec4b06ba3c02f/cfme/tests/containers/test_projects_dashboard.py#L12

Reply:
setup_provider fixture works on top of provider fixture. You provider fixture has scope="function", what means that provider has to be created and removed for every test where provider fixture is present.

This is rather bug that your provider is present in test where provider fixture isn't present.

So, I think you should either add provider fixture to test or change scope to module for mark.provider.
Second is preferable because it won't make pytest add and remove provider for every test in file.

Reply:
@izapolsk I think I got it, I added ```setup_provider```